### DISCUSSION TITLE
怎样接着训练 epoch9 10 11?

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#160] [General]
怎样接着训练 epoch9 10 11?
loss 不同

**Comment by Senhui-Qiu** [comment]
How to continue training epoch 10? Have you solved this problem yet?

**Comment by ZhiyuanChen** [comment]
我无法理解你的问题。
请提供更多的细节。

**Comment by ZhiyuanChen** [comment]
由于本issue没有得到更进一步的提问，我现在将其关闭。
如有新的问题，请重新打开它或者创建一个新的issue。
